go.mod

```diff
@@ -4,7 +4,7 @@ go 1.17
 
 require (
 	github.com/mitchellh/mapstructure v1.4.1
-	github.com/multiversx/mx-chain-core-go v1.1.35-0.20230302100006-5230818062e9
+	github.com/multiversx/mx-chain-core-go v1.2.1-0.20230404073242-a0f27673126a
```

Comment:
proper release

Reply:
Will do before the whole feature is merged.